src/core/util/logger.js

```diff
@@ -216,26 +209,30 @@ class Logger {
     }
 }
 
- /**
-  * Creates and returns a Context object based on some json data and jsonata
-  * query.
-  *
-  * @param {*} input - JSON object.
-  * @param {string} jna - jsonata query string.
-  * @returns a newly created {@link module:logger~Context} object.
-  */
+/**
+ * Creates and returns a Context object based on some json data and jsonata
+ * query.
+ *
+ * @param {*} input - JSON object.
+ * @param {string} jna - jsonata query string.
+ * @returns a newly created {@link module:logger~Context} object.
+ */
 Logger.createContext = function createContext(input, jna) {
     return new Context(input, jna);
```

Comment:
it looks like the constructor with 2 arguments was removed with this change ?

Reply:
Thank you for the hint. Fixed the documentation.